tox.ini

```diff
@@ -164,9 +164,20 @@ deps=
       pydocstyle
       pygments
       black==21.12b0
+      mypy
+      slotscheck>=0.12,<0.13
 commands =
      flake8 ./lib/ ./test/ ./examples/ setup.py doc/build/conf.py {posargs}
      black --check ./lib/ ./test/ ./examples/ setup.py doc/build/conf.py
+     slotscheck -m sqlalchemy
+
+
+# "pep8" env was renamed to "lint".
+# Kept for backwards compatibility until rename is completed elsewhere.
+[testenv:pep8]
+basepython = {[testenv:lint]basepython}
+deps = {[testenv:lint]deps}
```

Comment:
**mike bayer** (zzzeek) wrote:

the lint action doesn't, problem solved

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3576